### ISSUE TITLE
[Documentation / Tutorial] Extend https://opalrb.com/docs/guides/v1.1.1/getting_started.html

### ISSUE THREAD

**Issue by rubyFeedback** [opal/opal#2228]
Hey folks,

I am using https://opalrb.com/docs/guides/v1.1.1/getting_started.html right now.

I know ruby fairly well, wrote tons of stuff, but I am literally an opal noob and my
javascript knowledge is like from 2005 at best. So I don't really understand 
javascript that well and I am wondering how to do things via opal.

I propose to extend that tutorial.

What would I like to see on that page?

(1) Perhaps consider adding a link to "continue" learning opal from there, with
a bit of advanced knowledge.

(2) Please confirm on that page that one is able to use a .rb file directly and
show the syntax too. Right now I know we can use Opal.compile() and I 
copy/pasted the content of a .rb file and at the least the output seems to
work. But I am not sure if it actually works, whether there are limitations
or not.

(3) Please also show a standalone example, perhaps ideally just a simple
.html file where that opal-code can be displayed to work. A simple hello
world should suffice but perhaps a simple alert() functionality or an
on-click functionality or anything. Or all of this. :)

The goal here is to simply get people boostrapped with opal faster. 
You can not imagine how much I struggle right now to "get into" opal ...
I am sure it is easier once you are there, see how things work, but right
now I lack that knowledge and it is VERY hard to get going...

**Comment by rubyFeedback** [comment]
(Note that this primarily refers to the given URL. That is not to say to not
extend other parts of the documentation; I just had to focus on one point
first, and since I clicked on the "getting started" part first, that was
the first entry point for me. There are also some external tutorials
such as the conway thing, but it was too complex for me. I kind of 
need the groundwork for the start.)

**Comment by hmdne** [comment]
I agree a lot that getting Opal to work is kind of hard. I struggled a lot and with this struggle I found out there were a lot of integrations missing and documentation is kind of spread apart. Anyway that's a solid argumentation and idea in general.

My proposal for solving this issue is this:

We are available on Opal Slack channel where we may support you in real time. Once you are up and going, you can open pull requests fixing the documentation (we will also be able to assist you in that). Are you maybe willing to dedicate some time to do that? :D
